.github/actions/setup-dfx/action.yml

```diff
@@ -22,7 +18,7 @@ runs:
           echo "DFX restored from cache"
         else
           echo "DFX not restored from cache, running install script:"
-          DFX_VERSION=${{ env.dfx_version }} sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
+          DFX_VERSION=0.14.2 sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
```

Comment:
I had copy pasted this from another project. The other project had a root `dfx.json` but response verification doesn't. For now I think it's fine to lock this down manually. But once we have this feature in Cargo: https://github.com/rust-lang/cargo/issues/9406 we should be able to include the DFX projects into the Cargo workspace and then create a single `dfx.json` file to handle all DFX projects and clean this up.